### DISCUSSION TITLE
✅ iphone Storage

### DISCUSSION THREAD

**Discussion by pegh2020** [isontheline/pro.webssh.net#405] [Q&A]
Hi!
Love the app!
Have been using it a great deal including the sftp feature.
However, when I look in the iphone storage section (settings) I see that it is using 6,41GB (Documents and Data). The app itself uses 40,5 MB.
Probably cache?
Any idea how to get rid of it?
Paul

**Comment by isontheline** [comment]
Hi @pegh2020 👋

Thank you a lot for your support 🙏

Are you using WebSSH 16.0?
The latest release should vacuum temporary files at startup.

If it's not the case, could you run these steps please :
* Launch WebSSH$
* Launch mashREPL
* `du -sh ../*`

It should show the directories sizes of WebSSH.

**Reply by pegh2020** [reply]
ok, thanks for the quick reply.
I wasn’t on the latest version (fixed now!)
Before I updated I found cached files in “tmp” (via meshRepl).
These were video files Id FTP’d into another app (nPlayer) and removed from nplayer after watching. I guess the apple file system creates a “link” for them (so as long as they both exist it doesn’t take up too much space). But removing them from nPlayer left them untouched in “tmp”.

After the upgrade though all seems well. They are removed and WebSSH is not in my top list for iphone storage usage anymore!! ;0)

Thanks once again! An awesome go to app!

Paul

**Reply by isontheline** [reply]
Thank you a lot for your feedback and support about WebSSH 🙏
Happy to hear that your issue is now resolved with the latest release.

Have a nice day Paul and thank you again☀️

Arnaud
